libs/dh/shared/feature-authorization/src/lib/dh-selected-actor.store.ts

```diff
@@ -57,16 +74,59 @@ export class DhSelectedActorStore extends ComponentStore<SelectedActorState> {
 
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
             const actor = r.find((x) => x.id === actorId)!;
+
+            const actors = r
+              .map((x) => ({
+                id: x.id,
+                gln: x.gln,
+                actorName: 'inc.',
+                organizationName: x.organizationName,
+                selected: x.id === actorId,
+              }))
+              .sort((a, b) => {
+                const nameCompare = a.organizationName.localeCompare(
+                  b.organizationName
+                );
+                return nameCompare !== 0
+                  ? nameCompare
+                  : a.gln.localeCompare(b.gln);
+              });
+
+            const actorGroups = Array.from(
+              this.groupBy(actors, 'organizationName').entries()
+            ).map((x) => ({ organizationName: x[0], actors: x[1] }));
+
             return this.patchState({
               isLoading: false,
+              actorGroups: actorGroups,
               selectedActor: {
+                id: actor.id,
                 gln: actor.gln,
+                actorName: 'inc.',
                 organizationName: actor.organizationName,
+                selected: actor.id === actorId,
               },
             });
           })
         )
       )
     );
   });
+
+  public setSelectedActor(actorId: string) {
+    this.actorStorage.setSelectedActor(actorId);
+    window.location.reload();
```

Comment:
suggestion: let's inject the `window.location` object as an Injection token similar to `actorStorageToken`. This way it will be way easier to mock the `reload` method for testing (place it in a separate file for now. It will be moved to a `util` library later).

```ts
export const windowLocationToken = new InjectionToken<Location>(
  'windowLocationToken',
  {
    factory: () => window.location,
  }
);
```

It can be injected:

```
@Inject(windowLocationToken) private windowLocation: Location
```

and used:

```ts
this.windowLocation.reload();
```